plugin/vcs/github/github.go

```diff
@@ -116,9 +117,62 @@ func (p *Provider) FetchRepositoryActiveMemberList(ctx context.Context, oauthCtx
 	return nil, errors.New("not implemented yet") // TODO: https://github.com/bytebase/bytebase/issues/928
 }
 
-// ExchangeOAuthToken exchange oauth content with the provided authentication code.
-func (p *Provider) ExchangeOAuthToken(ctx context.Context, instanceURL string, oauthExchange *common.OAuthExchange) (*vcs.OAuthToken, error) {
-	return nil, errors.New("not implemented yet")
+// oauthResponse is a GitHub OAuth response.
+type oauthResponse struct {
+	AccessToken      string `json:"access_token" `
+	Error            string `json:"error,omitempty"`
+	ErrorDescription string `json:"error_description,omitempty"`
+	ErrorURI         string `json:"error_uri,omitempty"`
```

Comment:
not see this field being used.

Reply:
Removed in https://github.com/bytebase/bytebase/pull/998/commits/2bbf9f9aa87976c36134cc2524a6b91ca1cd66eb.